docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+:orphan:
+:nosearch:
+
+Working with Tabular Data
+=========================
+
+Motivation 
+----------
+
+In the following examples, we will use real data to demonstrate the utility of OpenDP functions.
+We will use the `Labour Force Survey microdata <https://ec.europa.eu/eurostat/web/microdata/public-microdata/labour-force-survey>`_ released by Eurostat for a few reasons: 
+
+1. **Generality:** The dataset is relatively general, with variables and contexts accessible to users across various domains.
+2. **Sample Utility:** The public microdata is a sample of the private, full microdata. Methods developed with the public microdata will also work on the private microdata, and researchers can request access to the full dataset through Eurostat. 
+3. **Realism**: Since the dataset tracks individuals over multiple years, we need to think more carefully about our unit of privacy.
+
+The specific methods that will be demonstrated are: 
+
+* Computing Fundemental Statistics 
+    * Sum 
+    * Mean 
+    * Median 
+    * Quantiles 
+* Aggregrations and Filtering 
+    * Grouping By Singular Variables
+    * Grouping By Multiple Variables 
+    * Filtering 
+* Using Known and Unknown Keys in Aggregrations
+* Data Preparation Limitations
```

Comment:
This is getting rendered in a funky way. Also, what do these two limitations mean? Might want to write a sentence about each.